app/src/main/java/org/mozilla/fenix/search/SearchController.kt

```diff
@@ -76,6 +76,7 @@ class DefaultSearchController(
     }
 
     override fun handleEditingCancelled() {
+        store.dispatch(SearchFragmentAction.EditingCancelled)
```

Comment:
Nit: there's nothing wrong with this name, but just to align with AC I'd prefer to name it `UpdateEditingCanceled`.